htdocs/include/comment_view.php

```diff
@@ -24,6 +24,8 @@
 
 include_once $GLOBALS['xoops']->path('include/comment_constants.php');
 
+global xoopsModuleConfig;
```

Comment:
That needs a $ or it is a hard error.

Not sure how that is de-referenced in your case, as it is fine as is in all the cases I've tried so far. Fix the missing dollar sign (and make sure it fixes your case, as this didn't :smile: ) then we can merge it.